src/System.Globalization.Calendars/tests/System/Globalization/KoreanLunisolarCalendarTests.cs

```diff
@@ -10,7 +10,7 @@ public class KoreanLunisolarCalendarTests : EastAsianLunisolarCalendarTestBase
     {
         public override Calendar Calendar => new KoreanLunisolarCalendar();
 
-        public override DateTime MinSupportedDateTime => new DateTime(0918, 02, 19);
+        public override DateTime MinSupportedDateTime => new DateTime(0918, 02, !PlatformDetection.IsFullFramework ? 19 : 14);
```

Comment:
According to the docs:

> The earliest date and time supported by the KoreanLunisolarCalendar class, which is equivalent to the first moment of February 14, 918 C.E. in the Gregorian calendar.

Is this a bug?

Reply:
We have done some fixes in this calendar in Net Core which included the min date change. we can file an issue to fix the doc in https://github.com/dotnet/docs